### ISSUE TITLE
"add -mforce-l32" patch seems to break "fix PR target/94584"

### ISSUE THREAD

**Issue by jjsuwa-sys3175** [earlephilhower/esp-quick-toolchain#35]
/cc @jcmvbkbc

with "add -mforce-l32" patch, `make check-c RUNTESTFLAGS="xtensa.exp"` fails:
```
Running /tmp/gcc-10.3.0-xtensa-lx106/gcc/testsuite/gcc.target/xtensa/xtensa.exp ...
FAIL: gcc.target/xtensa/pr94584.c scan-assembler-times memw 4
```
https://github.com/earlephilhower/esp-quick-toolchain/blob/master/patches/gcc10.3/gcc-mforce-l32.patch#L170
i guess `l8ui\t%0, %1` should prepend `%v1`

**Comment by jcmvbkbc** [comment]
> i guess l8ui\t%0, %1 should prepend %v1

Yes, I agree. Also I guess that %v1 should be in the following line, right before the l32i, and in similar place in line 154.
